### ISSUE TITLE
CORS not working

### ISSUE THREAD

**Issue by SeanWong24** [alosaur/alosaur#14]
I tried to enable CORS middleware by putting this into my code.
```ts
app.useCors(
  new CorsBuilder()
    .WithOrigins("*")
    .AllowAnyMethod()
    .AllowAnyHeaders()
);
```

But it does not working properly and it does not respond the OPTIONS request sent by the browser (got 404).

**Comment by irustm** [comment]
Thanks!
OPTIONS request was fixed in v0.6.0

**Comment by SeanWong24** [comment]
Thanks for the fix. After I pull the latest version, it responds the OPTIONS request now, but I still got blocked by the CORS policy. It seems working fine for GET and POST but not for PUT and DELETE.

**Comment by irustm** [comment]
Fixed v0.6.1. Try it please

**Comment by SeanWong24** [comment]
It works for me now. Thanks!
